implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontMeterRegistry.java

```diff
@@ -183,10 +237,19 @@ private boolean directToApi() {
         final Stream.Builder<String> metrics = Stream.builder();
 
         Meter.Id id = summary.getId();
-        addMetric(metrics, id, "sum", wallTime, summary.totalAmount());
-        addMetric(metrics, id, "count", wallTime, summary.count());
-        addMetric(metrics, id, "avg", wallTime, summary.mean());
-        addMetric(metrics, id, "max", wallTime, summary.max());
+
+        if (summary instanceof WavefrontHistogram) {
+            if (wavefrontSender != null) {
+                sendWavefrontHistogram(id, ((WavefrontHistogram) summary).flushDistributions());
```

Comment:
Are we only using the SDK to send histograms? Seems like we should send either all metrics or none through the SDK. If the former, need to be sure that the SDK supports injecting something like `HttpSender`.

Reply:
If a `WavefrontSender` is specified, the SDK is used to send all metrics and histograms.  If not specified, we're still using the old method (`HttpSender`, etc) for backwards compatibility, but the old method doesn't support the sending of wfhistograms (due to missing config options).

Reply:
We need to collapse these two concepts I think. Having to choose between being able to use (for example) a proxy and being able to send a histogram is not a choice we want our users to have to be required to make. I'd like to see an update to the wavefront-sdk to permit sender injection, or that at least for now, we persist in sending all data directly to API.